src/site/_data/i18n/footer.yml

```diff
@@ -6,9 +6,12 @@ view_source:
   en: View source
   es: Ver fuente
 
-updates:
-  en: Updates
-  es: Actualizaciones
+related_content:
+  en: Related content
```

Comment:
Perhaps for Spanish `Contenido relevante`

Reply:
Gracias!